mir-ci/mir_ci/robot_resources/screencopy.resource

```diff
@@ -8,93 +9,161 @@ Library             ../robot_libraries/WaylandHid.py
-    ${point}=    Evaluate    {"x": ${point}[x] + ${x}, "y": ${point}[y] + ${y}}
+    [Documentation]    Shift a point by the specified displacements along
+    ...    the x and y axes.
+    ...
+    ...    Arguments:
+    ...    - ${point}: Point to be displaced, as a tuple (x, y).
+    ...    - ${x}: Displacement along the x-axis.
+    ...    - ${y}: Displacement along the y-axis.
+    ...
+    ...    Return:
+    ...    Displaced point, as a tuple (x, y) of integers.
+    ${point}=    Evaluate    (int(${point}[0] + ${x}), int(${point}[1] + ${y}))
     RETURN    ${point}
 
-Move Pointer To ${destination}
-    [Documentation]    Move the pointer to a position or image template.
+Move Pointer To (${x}, ${y})
+    [Documentation]    Move the pointer to an absolute position.
+    ...
+    ...    Arguments:
+    ...    - ${x}: Integer absolute x-coordinate to move the pointer to.
+    ...    - ${y}: Integer absolute y-coordinate to move the pointer to.
+    WaylandHid.Move Pointer To Absolute    ${x}    ${y}
+
+Move Pointer To ${destination:^\(?!\(.+,.+\)\).+}
+    [Documentation]    Move the pointer to an absolute position or image
+    ...    template.
     ...
-    ...    When ${destination} is the path of an image template file, the pointer
-    ...    will move to the center of the first matching template region.
+    ...    Arguments:
+    ...    - ${destination}: Where to move the pointer to. If ${destination}
+    ...      is a tuple (x, y) of integers, the pointer will move to the
+    ...      absolute position given by the tuple. Otherwise, if
+    ...      ${destination} is the path of an image template file, the pointer
+    ...      will move to the center of the first matching template region.
     ...
-    ...    The return value is the pointer position after the move.
+    ...    Return:
+    ...    Absolute position of the pointer after the move, as  a tuple (x, y)
+    ...    of integers.
     IF    ${{isinstance($destination, str)}}
         ${regions}=    Screencopy.Match    ${destination}
         ${position}=    Get Center Of ${regions}[0]
     ELSE
         ${position}=    Set Variable    ${destination}
     END
-    WaylandHid.Move Pointer To Absolute    ${position}[x]    ${position}[y]
+    WaylandHid.Move Pointer To Absolute    ${position}[0]    ${position}[1]
     RETURN    ${position}
 
-Move Pointer To Proportional ${position}
-    [Documentation]    Move the pointer to a position given as proportions to
-    ...    the size of the display output.
+Move Pointer To Proportional (${x}, ${y})
+    [Documentation]    Move the pointer to a destination position given as
+    ...    proportions to the size of the display output.
     ...
-    ...    ${position} must be a point represented by floating-point x and y
-    ...    values in the range 0..1.
+    ...    Arguments:
+    ...    - ${x}: Relative x-coordinate to move the pointer to. It must be in
```

Comment:
"Output-relative"? It's not clear what it's relative to.

Reply:
Yes, changed to "output-relative".